atom/browser/common_web_contents_delegate.cc

```diff
@@ -181,10 +181,11 @@ void CommonWebContentsDelegate::SetOwnerWindow(
     content::WebContents* web_contents, NativeWindow* owner_window) {
   owner_window_ = owner_window ? owner_window->GetWeakPtr() : nullptr;
   auto relay = base::MakeUnique<NativeWindowRelay>(owner_window_);
+  auto relayKey = relay->key;
```

Comment:
According to the Style Guide the name should be `relay_key`.
https://google.github.io/styleguide/cppguide.html#Variable_Names